src/views/SidebarView.vue

```diff
@@ -64,11 +64,11 @@ import AgeFacet from '../components/facets/AgeFacet.vue'
 import RangeFacet from '../components/facets/RangeFacet.vue'
 import { mapMutations } from 'vuex'
 import CountView from '@/views/CountView'
-import ClickOutside from 'vue-click-outside'
+import ClickOutside from 'v-click-outside'
 import { library } from '@fortawesome/fontawesome-svg-core'
-import { faMinusCircle } from '@fortawesome/free-solid-svg-icons'
+import { faAngleDoubleLeft, faAngleDoubleDown } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
-library.add(faMinusCircle)
+library.add(faAngleDoubleLeft, faAngleDoubleDown)
```

Comment:
bit confused how this relates to IE11

Reply:
moved to:
https://github.com/molgenis/molgenis-app-lifelines-webshop/pull/92